docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -55,21 +55,39 @@ Your Bitbucket integration is now complete. Pulse will start loading your data f
 
 -   Pulse considers a deployment every merged pull request that **targets the default branch** of the repository.
+
 -   The deployment date is the timestamp when the corresponding pull request is merged.
+
 -   The set of changes in a deployment is the list of commits in the corresponding pull request.
+
 -   Pulse associates all the Bitbucket user groups of the author of a merged pull request with the corresponding deployment, excluding the user groups with less than two members. Pulse only takes the changes to Bitbucket user groups into account on pull requests merged after those changes.
 
 ## Automatic incident detection strategies {: id="incident-detection-strategy"}
 
-The Pulse Bitbucket integration detects incidents automatically using [pull request reverts](#bb-incident-pr-revert). You can also choose [not to detect incidents via Bitbucket](#bb-incident-not-detect).
+The Pulse Bitbucket integration can detect **incidents** automatically using the following strategies:
+
+-   [Pull request reverts](#bb-incident-pr-revert)
+-   [Pull request naming convention](#bb-incident-naming)
+
+You can also choose [not to detect incidents via Bitbucket](#bb-incident-not-detect).
 
 ### Use pull request reverts (based on default branch) {: id="bb-incident-pr-revert"}
 
 -   Pulse bases incident detection on [pull request reverts](https://support.atlassian.com/bitbucket-cloud/docs/merge-a-pull-request/#Revert-a-merged-pull-request).
+
 -   Pulse considers an incident any pull request that **targets the default branch** of the repository merged from a branch whose name starts with `revert-pr-`, getting the number of the reverted pull request from the branch name, `revert-pr-<pull request number>`. If you change the name of the branch created by Bitbucket when you revert a pull request, Pulse may not be able to obtain the incident data correctly.
+
 -   The incident creation date is the timestamp when the reverted pull request was initially merged. If Pulse can't get the reverted pull request number from the branch name, the incident creation date is the timestamp of the first commit to the incident pull request.
 -   Pulse associates incidents to the system matching the repository name.
 
+### Use pull request naming convention (based on default branch) {: id="bb-incident-naming"}
+
+-   Pulse applies the same mechanism to detect incidents used for the [pull request reverts](#bb-incident-pr-revert) strategy. However, instead of considering branches whose name starts with `revert-pr-`, Pulse uses the prefix that you define for the **branch name** or the **pull request title** to match the naming convention of your workflow.
+
+    ![Pulse custom incident detection](images/bitbucket-incident-naming-convention.png)
```

Comment:
clean: We could omit this screenshot here and update the main screenshot `bitbucket-incident-strategy.png` to have the naming convention option selected and showing the available configurations.

Reply:
Thank you for this input, @prcr, but I don't completely agree with this suggestion... 🤔 
I believe the reader will benefit from having this screenshot near the details for this strategy, where we describe how Pulse uses the prefix and its limitations. Having a such specific screenshot in the setup flow without the details would be more confusing, IMO.
WDYT?

Reply:
That's a sound argument :+1:, feel free to disregard my suggestion.